### PR TITLE
QLog: v1: Data Transformation: CPLs: Update moviebuff-sdk-go to get Languages

### PR DESCRIPTION
**Pull request by ashwinspg**
### JIRA Link:
[QL-63](https://qubecinema.atlassian.net/browse/QL-63?atlOrigin=eyJpIjoiOGM5NWQzNzNmNzAxNDY1MjhkNjAwYTY2NDA0MmYwNjAiLCJwIjoiaiJ9)

### CODE REVIEW

File: v2/languages.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package moviebuff
+
+type Language struct {
+	UUID       string   `json:"uuid"`
+	Name       string   `json:"name"`
+	ISO639_1   string   `json:"iso639_1"`
+	ISO639_2   string   `json:"iso639_2"`
+	IANA       string   `json:"iana"`
+	NativeName string   `json:"native_name"`
+	SpokedIn   []string `json:"spoken_in"`
```
Comment (svetha-cvl) [inline v2/languages.go:10]:
```suggestion
	SpokenIn   []string `json:"spoken_in"`
```

File: v2/moviebuff.go [hunk 2]
```diff
@@ -335,3 +336,41 @@ func (m *moviebuff) GetHolidayCalendar(ctx context.Context, countryID string) (*
...
+
+	case http.StatusOK:
+		content, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return nil, err
+		}
+
+		var languages []Language
+		err = json.Unmarshal(content, &languages)
+		if err != nil {
```
Comment (svetha-cvl) [inline v2/moviebuff.go:367]:
```suggestion
		return languages, err
```

Comment (suryapandian) [reply v2/moviebuff.go:367]: Yeah if possible, for the prev function, too in L327, we could simply `return calendarInfo, nil`

**Review by suryapandian — COMMENTED** [review]
LGTM Otherwise

File: v2/moviebuff.go (hunk 2 above)
Comment (ashwinspg) [reply v2/moviebuff.go:367]: We can't do the same approach for `GetHolidayCalendar` function as it needs to return nil when there is a error. But, I'll do the change in `GetCertifications` function.

Comment (suryapandian) [reply v2/moviebuff.go]: I think we could still return with `calendarInfo`, but clients could be expecting nil when there is no data 🤔 
Anyways, that is additional things, if we are ambiguous we can ignore :)

**Review by suryapandian — APPROVED** [review]

**Review by svetha-cvl — APPROVED** [review]

**Review by sasipriyags — APPROVED** [review]

**Pull request merged**